### ISSUE TITLE
Is there no client ip in the access log?

### ISSUE THREAD

**Issue by Dup4** [hertz-contrib/logger#23]
https://github.com/hertz-contrib/logger/blob/2de215a9fd505da955c4646f4fe9fd3c6d5bcff5/accesslog/tag.go#L91-L98

https://github.com/hertz-contrib/logger/blob/2de215a9fd505da955c4646f4fe9fd3c6d5bcff5/accesslog/tag.go#L102-L104

I noticed that there is no `client_ip` field in accesslog, among the relevant fields are `ip`, `ips`, `host`, where `ip` is the IP of the server address and `host` is the server address, but may contain `port`.

`ips` is `X-Forwarded-For`, but if the user accesses it directly and it is not forwarded, the `X-Forwarded-For` header may not be set, so it may be empty.

So in summary, there is no `client_ip` in the access_log at the moment, have we considered adding a field for this? Maybe we could take it from `c.ClientIP()`.

**Comment by FGYFFFF** [comment]
This is a good idea, can you contribute code?

**Comment by Dup4** [comment]
> This is a good idea, can you contribute code?

OK, I can try to give a PR.

**Comment by li-jin-gou** [comment]
cool!
